Products/PleiadesEntity/browser/adapters/place.py

```diff
@@ -17,10 +17,10 @@
 class PlaceExportAdapter(WorkExportAdapter, ContentExportAdapter):
 
     def _connectsWith(self):
-        return self.brain.connectsWith or []
+        return getattr(self.brain, 'connectsWith', None) or []
 
     def _hasConnectionsWith(self):
-        return self.brain.hasConnectionsWith or []
+        return getattr(self.brain, 'hasConnectionsWith', None) or []
```

Comment:
Just FYI, this change is unrelated but I added this to fix an error my local dev instance when viewing places.